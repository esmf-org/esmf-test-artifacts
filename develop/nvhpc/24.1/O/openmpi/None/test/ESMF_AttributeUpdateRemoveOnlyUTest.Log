2025-05-08 07:52:38
20250508 083402.655 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.655 INFO             PET0 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250508 083402.655 INFO             PET0 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250508 083402.655 INFO             PET0 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250508 083402.655 INFO             PET0 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250508 083402.655 INFO             PET0 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.655 INFO             PET0 Running with ESMF Version   : v8.9.0b08-5-g079ca68fdc
20250508 083402.655 INFO             PET0 ESMF library build date/time: "May  8 2025" "07:49:07"
20250508 083402.655 INFO             PET0 ESMF library build location : /scratch2/NCEPDEV/stmp1/Gerhard.Theurich/ESMF-Nightly-Testing/nvhpc_24.1_openmpi_O_develop/esmf
20250508 083402.655 INFO             PET0 ESMF_COMM                   : openmpi
20250508 083402.656 INFO             PET0 ESMF_MOAB                   : enabled
20250508 083402.656 INFO             PET0 ESMF_LAPACK                 : enabled
20250508 083402.656 INFO             PET0 ESMF_NETCDF                 : disabled
20250508 083402.656 INFO             PET0 ESMF_PNETCDF                : disabled
20250508 083402.656 INFO             PET0 ESMF_PIO                    : disabled
20250508 083402.656 INFO             PET0 ESMF_YAMLCPP                : enabled
20250508 083402.680 INFO             PET0  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
20250508 083402.680 INFO             PET0  NUMBER_OF_PROCESSORS            4
20250508 083402.685 INFO             PET0  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
20250508 083402.685 ERROR            PET0 ESMF_Grid.F90:17075 ESMF_GridDestroy Object being used before creation  - Bad Object
20250508 083402.685 INFO             PET0  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
20250508 083402.685 INFO             PET0  Finalizing ESMF
20250508 083402.656 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.656 INFO             PET1 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250508 083402.656 INFO             PET1 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250508 083402.656 INFO             PET1 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250508 083402.656 INFO             PET1 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250508 083402.656 INFO             PET1 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.656 INFO             PET1 Running with ESMF Version   : v8.9.0b08-5-g079ca68fdc
20250508 083402.656 INFO             PET1 ESMF library build date/time: "May  8 2025" "07:49:07"
20250508 083402.656 INFO             PET1 ESMF library build location : /scratch2/NCEPDEV/stmp1/Gerhard.Theurich/ESMF-Nightly-Testing/nvhpc_24.1_openmpi_O_develop/esmf
20250508 083402.656 INFO             PET1 ESMF_COMM                   : openmpi
20250508 083402.656 INFO             PET1 ESMF_MOAB                   : enabled
20250508 083402.656 INFO             PET1 ESMF_LAPACK                 : enabled
20250508 083402.656 INFO             PET1 ESMF_NETCDF                 : disabled
20250508 083402.656 INFO             PET1 ESMF_PNETCDF                : disabled
20250508 083402.656 INFO             PET1 ESMF_PIO                    : disabled
20250508 083402.656 INFO             PET1 ESMF_YAMLCPP                : enabled
20250508 083402.680 INFO             PET1  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
20250508 083402.680 INFO             PET1  NUMBER_OF_PROCESSORS            4
20250508 083402.685 INFO             PET1  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
20250508 083402.686 ERROR            PET1 ESMF_Grid.F90:17075 ESMF_GridDestroy Object being used before creation  - Bad Object
20250508 083402.686 INFO             PET1  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
20250508 083402.686 INFO             PET1  Finalizing ESMF
20250508 083402.655 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.655 INFO             PET2 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250508 083402.655 INFO             PET2 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250508 083402.655 INFO             PET2 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250508 083402.655 INFO             PET2 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250508 083402.655 INFO             PET2 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.655 INFO             PET2 Running with ESMF Version   : v8.9.0b08-5-g079ca68fdc
20250508 083402.655 INFO             PET2 ESMF library build date/time: "May  8 2025" "07:49:07"
20250508 083402.655 INFO             PET2 ESMF library build location : /scratch2/NCEPDEV/stmp1/Gerhard.Theurich/ESMF-Nightly-Testing/nvhpc_24.1_openmpi_O_develop/esmf
20250508 083402.655 INFO             PET2 ESMF_COMM                   : openmpi
20250508 083402.656 INFO             PET2 ESMF_MOAB                   : enabled
20250508 083402.656 INFO             PET2 ESMF_LAPACK                 : enabled
20250508 083402.656 INFO             PET2 ESMF_NETCDF                 : disabled
20250508 083402.656 INFO             PET2 ESMF_PNETCDF                : disabled
20250508 083402.656 INFO             PET2 ESMF_PIO                    : disabled
20250508 083402.656 INFO             PET2 ESMF_YAMLCPP                : enabled
20250508 083402.680 INFO             PET2  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
20250508 083402.680 INFO             PET2  NUMBER_OF_PROCESSORS            4
20250508 083402.685 INFO             PET2  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
20250508 083402.685 INFO             PET2  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
20250508 083402.685 INFO             PET2  Finalizing ESMF
20250508 083402.679 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.679 INFO             PET3 !!! THE ESMF_LOG IS SET TO OUTPUT ALL LOG MESSAGES !!!
20250508 083402.679 INFO             PET3 !!!     THIS MAY CAUSE SLOWDOWN IN PERFORMANCE     !!!
20250508 083402.679 INFO             PET3 !!! FOR PRODUCTION RUNS, USE:                      !!!
20250508 083402.679 INFO             PET3 !!!                   ESMF_LOGKIND_Multi_On_Error  !!!
20250508 083402.679 INFO             PET3 !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
20250508 083402.679 INFO             PET3 Running with ESMF Version   : v8.9.0b08-5-g079ca68fdc
20250508 083402.679 INFO             PET3 ESMF library build date/time: "May  8 2025" "07:49:07"
20250508 083402.679 INFO             PET3 ESMF library build location : /scratch2/NCEPDEV/stmp1/Gerhard.Theurich/ESMF-Nightly-Testing/nvhpc_24.1_openmpi_O_develop/esmf
20250508 083402.679 INFO             PET3 ESMF_COMM                   : openmpi
20250508 083402.680 INFO             PET3 ESMF_MOAB                   : enabled
20250508 083402.680 INFO             PET3 ESMF_LAPACK                 : enabled
20250508 083402.680 INFO             PET3 ESMF_NETCDF                 : disabled
20250508 083402.680 INFO             PET3 ESMF_PNETCDF                : disabled
20250508 083402.680 INFO             PET3 ESMF_PIO                    : disabled
20250508 083402.680 INFO             PET3 ESMF_YAMLCPP                : enabled
20250508 083402.680 INFO             PET3  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
20250508 083402.680 INFO             PET3  NUMBER_OF_PROCESSORS            4
20250508 083402.685 INFO             PET3  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
20250508 083402.686 INFO             PET3  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
20250508 083402.686 INFO             PET3  Finalizing ESMF
