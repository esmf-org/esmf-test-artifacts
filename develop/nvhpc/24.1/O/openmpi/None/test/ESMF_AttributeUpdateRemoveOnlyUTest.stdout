2025-05-08 07:52:38
  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
  NUMBER_OF_PROCESSORS            4
  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
  NUMBER_OF_PROCESSORS            4
  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
  NUMBER_OF_PROCESSORS            4
  Beginning Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 533
  NUMBER_OF_PROCESSORS            4
  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
  PASS  Getting an updated deleted Attribute value from a Field test, ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 621
  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
  Ending Test, file ESMF_AttributeUpdateRemoveOnlyUTest.F90, line 646
  PET            3  Test Elapsed Time     59.04698371887207       msec.
  PET            1  Test Elapsed Time     59.05199050903320       msec.
  PET            0  Test Elapsed Time     59.51809883117676       msec.
  PET            2  Test Elapsed Time     59.49807167053223       msec.
